src/listing--rating-panel.html

```diff
@@ -21,7 +22,36 @@
         <![endif]-->
 
         <main class="container">
-        <!-- Add your site or application content here -->
+          <section class="reviews">
+          <ul class="panel-rating">
+            <li class="review">
+              <section class="first-column-review">
+                <a class="im-buyer"href="#buyer">
+                  <img src="https://img0.etsystatic.com/100/0/72350745/iusa_75x75.34516074_9qam.jpg" alt="" />
+                     <!-- {{img from buyer}} -->
+                </a>
+                <p class="review-from">Reviewed by
+                  <a href="#buyer">{{buyer name}}</a>
+                </p>
+             </section><!-- .end of section first-column-review -->
+             <section class="second-column-review">
+                <p class="review-date">{{review date}}</p>
+                <div class="review-starts">
+                <div class="rating">
+                    <span>☆</span><span>☆</span><span>☆</span><span>☆</span><span>☆</span>
+                </div>
+                <section class="item-im-descr">
+                  <a class="item-img" href="#{{item-img}}"><img src="https://img0.etsystatic.com/068/0/6135474/il_75x75.767391102_mwt1.jpg" alt="" /></a>
+                  <a class="item-descr" href="#{{item-descr}}">{{item description}}</a>
+                </section>
+
+
+
+              </section>
+            </li>
+            <li class="review"></li>
+          </ul>
+        </section><!-- .end section reviews -->
```

Comment:
Finally, I noticed that you _moved_ `index.html` to `listing--rating-panel.html` instead of _copying_ it. That's gonna create some problems for us when merging it. See me to undo that step without losing your work.